server/app/services/openApi/v2/Info.java

```diff
@@ -0,0 +1,53 @@
+package services.openApi.v2;
+
+import com.google.auto.value.AutoValue;
+import java.util.Optional;
+
+/**
+ * The object provides metadata about the API. The metadata can be used by the clients if needed,
+ * and can be presented in the Swagger-UI for convenience.
+ *
+ * <p>https://swagger.io/specification/v2/#info-object
+ */
+@AutoValue
+public abstract class Info {
+  /**
+   * Required.
+   *
+   * <p>The title of the application.
+   */
+  public abstract String getTitle();
+
+  /**
+   * Required.
+   *
+   * <p>Provides the version of the application API (not to be confused with the specification
+   * version).
+   */
+  public abstract String getVersion();
+
+  /**
+   * A short description of the application. GFM syntax can be used for rich text representation.
```

Comment:
nit: can we spell this out. It's "Github-Flavored Markdown" right?

Reply:
The comments are copied from the official spec. I don't really want to edit them because then you can't just copy and paste the search string in the spec.

@avaleske  How do you feel about if I define it as a separate sentence at the end of the current string and five other instances in other classes?

Reply:
In retrospect, I think it's fine as is. I figured it out from context.